### PR TITLE
Create unit tests for back end

### PR DESCRIPTION
**Pull request by bat3**

### CODE REVIEW

File: business/tests/Services/PacsMirrorServiceUnitTests.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+            mockPacsService.Setup(repo => repo.GetSeriesAsync())
+                .ReturnsAsync(GetTestPacsSeries(4));
+
+            IPacsMirrorService pacsMirrorService = new PacsMirrorService(
+                mockSeriesServices.Object, 
+                mockPacsService.Object);
+            // Act
+            pacsMirrorService.MirrorPacs();
+            // Assert
+            Assert.AreEqual(true, true);
```
Comment (cquennouelle) [inline business/tests/Services/PacsMirrorServiceUnitTests.cs]: I don't get this assert.

File: business/tests/Services/PacsMirrorServiceUnitTests.cs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            mockPacsService.Setup(repo => repo.GetSeriesAsync())
+                .ReturnsAsync(GetTestPacsSeries(4));
+
+            IPacsMirrorService pacsMirrorService = new PacsMirrorService(
+                mockSeriesServices.Object,
+                mockPacsService.Object);
+            // Act
+            pacsMirrorService.MirrorPacs();
+            // Assert
+            Assert.AreEqual(true, true);
```
Comment (cquennouelle) [inline business/tests/Services/PacsMirrorServiceUnitTests.cs]: Idem, I don't understand the use of this assert.

File: business/tests/UnitTests.csproj [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <AssemblyName>UnitTests</AssemblyName>
+
+    <RootNamespace>UnitTests</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />
+    <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="NUnit" Version="3.12.0" />
```
Comment (cquennouelle) [inline business/tests/UnitTests.csproj:19]: Funny, you're using NUnit, while I've just started a project with XUnit.

**Review by cquennouelle — DISMISSED** [review]

File: business/tests/UnitTests.csproj (hunk 3 above)
Comment (bat3) [reply business/tests/UnitTests.csproj:19]: Yes, I create this tests before your merge request. But i don't like a framework who call test 'Fact'...
We can do a benchmark 🤪

File: business/tests/Services/PacsMirrorServiceUnitTests.cs (hunk 2 above)
Comment (bat3) [reply business/tests/Services/PacsMirrorServiceUnitTests.cs]: To delete ;-)

File: business/tests/Services/PacsMirrorServiceUnitTests.cs (hunk 1 above)
Comment (bat3) [reply business/tests/Services/PacsMirrorServiceUnitTests.cs]: To delete ;-)

**Review by cquennouelle — APPROVED** [review]

**Pull request merged**